components/member-profile/contribution/index.js

```diff
@@ -10,7 +12,51 @@ const renderPRLinks = (prList) =>
 
 const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
   const {
-    task: { title, startedOn, endsOn, status, purpose, featureUrl },
+    task: { featureUrl },
+    prList
+  } = contribution;
+  const url = featureUrl || prList[0]['url'];
+  const gotoUrl = () => url && window.open(url, '_blank');
+  const urlObj = url && URL.parse(url);
+
+  return (
+    <div>
+      {urlObj?.host === 'members.realdevsquad.com' ? (
+        <Link href={urlObj.path}>
+          <div className={url && classNames.contributionCard}>
+            <ContributionCard
+              contribution={contribution}
+              fullName={fullName}
+              imageLink={imageLink}
+              devUser={devUser}
+              url={url}
+              urlObj={urlObj}
+            />
+          </div>
+        </Link>
+      ) : (
+        <div
+          className={url && classNames.contributionCard}
+          onClick={gotoUrl}
+          onKeyDown={gotoUrl}
+          aria-hidden="true">
+          <ContributionCard
+            contribution={contribution}
+            fullName={fullName}
+            imageLink={imageLink}
+            devUser={devUser}
+            url={url}
+            urlObj={urlObj}
+          />
+        </div>
+      )}
+    </div>
```

Comment:
Nit: Can we decrease the repeated lines here somehow?

Duplicated code can lead to software that is hard to understand and difficult to change. The Don't Repeat Yourself (DRY) principle states:

Every piece of knowledge must have a single, unambiguous, authoritative representation within a system.

When you violate DRY, bugs and maintenance problems are sure to follow. Duplicated code has a tendency to both continue to replicate and also to diverge (leaving bugs as two similar implementations differ in subtle ways).

Reply:
@ankushdharkar made the changes checkout the last commit